src/main/java/blue/lapis/pore/impl/entity/PorePlayer.java

```diff
@@ -234,17 +253,15 @@ public void playNote(Location loc, Instrument instrument, Note note) {
 
     @Override
     public void playSound(Location location, Sound sound, float volume, float pitch) {
-        getHandle().playSound(SoundConverter.of(sound), VectorConverter.create3d(location), (double) volume,
-                (double) pitch);
+        getHandle().playSound(SoundConverter.of(sound), VectorConverter.create3d(location), volume, pitch);
     }
 
     @Override
     public void playSound(Location location, String sound, float volume, float pitch) {
         // TODO: Isn't the String sound the ID and not the name?
-        Optional<SoundType> spongeSound = Pore.getGame().getRegistry().getSound(sound);
+        Optional<SoundType> spongeSound = Pore.getGame().getRegistry().getType(SoundType.class, sound); //TODO: no idea whether this is right
```

Comment:
Based on the previous TODO, this might require bit of mapping, but we can worry about that later if it turns out to be broken, so no worries.